pkg/cmd/openshift-controller-manager/controller/unidling.go

```diff
@@ -10,11 +10,10 @@ import (
 	unidlingcontroller "github.com/openshift/origin/pkg/unidling/controller"
 )
 
-type UnidlingControllerConfig struct {
-	ResyncPeriod time.Duration
-}
+func RunUnidlingController(ctx ControllerContext) (bool, error) {
+	// TODO this should be configurable
```

Comment:
why not make it configurable now?

Reply:
> why not make it configurable now?

because this pull is already large and we didn't allow it before.